src/jdk.incubator.jextract/share/classes/jdk/internal/jextract/impl/StructBuilder.java

```diff
@@ -147,11 +151,11 @@ public void addVar(String javaName, String nativeName, MemoryLayout layout, Clas
     private void emitFieldGetter(Constant vhConstant, String javaName, Class<?> type) {
         incrAlign();
         indent();
-        append(MEMBER_MODS + " " + type.getSimpleName() + " " + javaName + "$get(MemorySegment seg) {\n");
+        append(MEMBER_MODS + " " + type.getSimpleName() + " " + javaName + "$get(MemorySegment " + safeParameterName("seg") + ") {\n");
```

Comment:
In this, and other methods, we could at least cache the safe name into a local and reuse it later?